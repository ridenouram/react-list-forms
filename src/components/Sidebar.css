.Container {
  display: flex;
  p {
    padding: 50px;
    
  }
}

.Sidebar {
  background: black;
  display: flex;
  flex-direction: column;
  width: 20em;
  a {
    color: white;
    font-size: 20px;
    padding: 20px;
  }
}
