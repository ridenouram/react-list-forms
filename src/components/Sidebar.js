import React from 'react';
import PropTypes from 'prop-types';
import styles from './Sidebar.css';

export default function Sidebar({ children }) {
  return (
    <nav className={styles.Sidebar}>
      {children}
    </nav>
  );
}

Sidebar.propTypes = {
  children: PropTypes.node
};
