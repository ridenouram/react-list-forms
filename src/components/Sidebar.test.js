import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('Sidebar component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Sidebar />);
    expect(wrapper).toMatchSnapshot();
  });
});
