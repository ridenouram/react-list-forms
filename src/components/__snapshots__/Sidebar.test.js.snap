// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component matches snapshot 1`] = `
<nav
  className="Sidebar"
/>
`;
